docs/sftt/sftt-api-spec-new.md

```diff
@@ -72,29 +80,51 @@ Must be called on a reservation belonging. Will cancel the reservation by settin
 This reservation was cancelled successfully.
 
 ##### `400 BAD REQUEST`
-If the reservation id specified is invalid.
+If the block id specified is invalid.
 
-### Delete a Reservation (Admin Only)
+### Uncomplete a Reservation (Admin Only)
 
-`POST api/v1/protected/reservations/delete`
+`POST api/v1/protected/reservations/uncomplete`
 
-Can only by called by admins. Will delete the specified reservation by setting deleted to `true`.
+Can only by called by admins. Will invalidate the last completion for the given block and opens the block back up for inventory.
 
 #### Request Body
 
 ```json
 {
-  "reservation": INT
+  "block_id": INT
 }
 ```
 
 #### Responses
 
 ##### `200 OK`
-This reservation was deleted successfully.
+This reservation was marked as incomplete successfully.
 
 ##### `400 BAD REQUEST`
-If the reservation id specified is invalid.
+If the block id specified is invalid.
+
+### Mark for QA (Admin Only)
+
+`POST api/v1/protected/reservations/qa`
+
+Can only by called by admins. Will indicate that this block needs QA. 
```

Comment:
Can this be called on a block in any state? If it's called on a completed block should the block count for the user that completed it or no?

Reply:
I think it should only be called on completed blocks, since otherwise there is not QA to do.

As for if the block counts as completed, I would say yes. If the block fails QA SFTT can mark the block as open again, which would lose the completion for the user/team

Reply:
Okay that makes sense. Let's make sure we specify that QA blocks still count somewhere, maybe in the leaderboard endpoints?

Reply:
Yes will do